akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -37,8 +37,8 @@ object GraphStages {
    * INERNAL API
    */
   private[stream] abstract class SimpleLinearGraphStage[T] extends GraphStage[FlowShape[T, T]] {
-    val in = Inlet[T]("in")
-    val out = Outlet[T]("out")
+    val in = Inlet[T](toString + ".in")
+    val out = Outlet[T](toString + ".out")
```

Comment:
Two string builders for every graph stage instance that inherits `SimpleLinearGraphStage`. Is there a more optimal  way to have properly named in(out)lets? Without this it is harder to follow debug output of fusing.